src/lib/util/dpr.ts

```diff
@@ -0,0 +1,12 @@
+export function getDPR(element: Element): number {
```

Comment:
Can we add some documentation at the top of the file explaining the purpose of each script. Feel free to link to the Floating UI source when relevant too. In case case I'm not sure what `DPR` is or does.

JSDoc comments on each function, etc would be helpful as well!